updates/9999_zooke_protecting_the_shipment.sql

```diff
@@ -0,0 +1,2 @@
+-- Fix quest 309 Protecting the Shipment completion point with the patched SD2 script
+UPDATE `mangos`.`quest_template` SET `SpecialFlags`='2' WHERE `entry`='309';
```

Comment:
pls remove the databasename `mangos`. not every user will name it's datbase like that :)